src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/tests/ViewRenderer.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import React from 'react';
+import ReactTestRenderer from 'react-test-renderer';
+import ViewRenderer from '../ViewRenderer';
+
+test('Render view returned from ViewRegistry', () => {
+    const ViewRegistry = jest.mock('../../../services/ViewRegistry', () => {
+        return {
+            getView: jest.fn(() => (<h1>Test</h1>)),
+        };
+    });
```

Comment:
Ah, thanks a lot, that helped. At least I am getting another error now, but I am looking into that on my own :-)